<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ejmdGeneralHintsWidget</class>
 <widget class="QWidget" name="ejmdGeneralHintsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>57</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="genHintsSubmissionLayout">
     <property name="margin">
      <number>4</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="genHintsCompilationLayout">
     <property name="margin">
      <number>4</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="genHintsExecutionLayout">
     <property name="margin">
      <number>4</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
